examples/webpack2-react-simple/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "webpack2-react-simple",
+  "main": "index.js",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "$(npm bin)/webpack"
```

Comment:
Just `webpack` should suffice here (NPM automatically sets the PATH to include the local npm bin directory).